grassroots-backend/src/GetEnvVars.ts

```diff
@@ -119,6 +126,35 @@ async function readSingleEnvironmentFile(
 // Cache environment variables.
 let envVariables: Environment | undefined = undefined;
 
+function logEnvVars(envVariables: Environment): void {
```

Comment:
Can I get a second pair of eyes on which should be sha1'd here?

Reply:
LGTM! I don't see anything else that looks sensitive.